tests/test_remote_sequential.py

```diff
@@ -118,7 +118,7 @@ def test_remote_sequential_prompts(batch_size=2, seq_len=5, pre_seq_len=3):
         block = load_pretrained_block(MODEL_NAME, block_index=block_index, torch_dtype=torch.float32)
         (outputs_ref,) = block(outputs_ref)
 
-    assert torch.allclose(outputs_ref, outputs)
+    assert torch.allclose(outputs_ref, outputs, atol=1e-5)
```

Comment:
This one *is* related to tensor-parallelism

Previously, the float32 no-compression server would run exactly the same sequence of operations and was exactly equal.
Now, we compare non-tensor-parallel model to a tensor-parallel one. It is still allcose, but the summation order slightly affects the outputs.